test/bitcrowd_ecto/repo_test.exs

```diff
@@ -20,9 +20,32 @@ defmodule BitcrowdEcto.RepoTest do
   describe "fetch/2" do
     setup [:insert_test_schema]
 
-    test "fetches a record by id and wraps it into an ok tuple", %{resource: resource} do
+    test "fetches a record by primary key and wraps it into an ok tuple", %{resource: resource} do
       assert TestRepo.fetch(TestSchema, resource.id) == {:ok, resource}
     end
+
```

Comment:
Please end the describe block starting line 20 here and start a new one, so we don't needlessly execute the setup helper. A bit nitpicky, but since the "not found" test ("fetch/2 when the resource does not exist") has its own describe block, we might as well follow this example.

```suggestion
  end
  describe "fetch/2 with schemas with non-standard` primary key"  do
```